dist/rpm/input-leap.spec.in

```diff
@@ -14,46 +14,44 @@ URL: https://github.com/input-leap/input-leap
 Source0: %{tarname}.tar.xz
 
 %if 0%{?fedora}
-BuildRequires: avahi-compat-libdns_sd-devel
-BuildRequires: cmake3
+BuildRequires: pkgconfig(avahi-compat-libdns_sd)
+BuildRequires: cmake3 >= 3
 BuildRequires: desktop-file-utils
 BuildRequires: gcc-c++
-BuildRequires: gmock-devel
-BuildRequires: gtest-devel
-BuildRequires: gulrak-filesystem-devel
-BuildRequires: libX11-devel
-BuildRequires: libXtst-devel libXinerama-devel libXrandr-devel
-BuildRequires: libICE-devel libSM-devel
-BuildRequires: libcurl-devel
-BuildRequires: openssl-devel
-BuildRequires: qt6-qtbase-devel
-BuildRequires: qt6-qttools-devel
-BuildRequires: qt6-qt5compat-devel
+BuildRequires: pkgconfig(gmock)
+BuildRequires: pkgconfig(gtest)
+BuildRequires: cmake(ghc_filesystem)
+BuildRequires: pkgconfig(x11)
+BuildRequires: pkgconfig(xtst) pkgconfig(xinerama) pkgconfig(xrandr)
+BuildRequires: pkgconfig(ice) pkgconfig(sm)
+BuildRequires: pkgconfig(libcurl)
+BuildRequires: pkgconfig(openssl)
+BuildRequires: pkgconfig(Qt6)
+BuildRequires: pkgconfig(Qt6Designer)
+BuildRequires: pkgconfig(Qt6Core5Compat)
 BuildRequires: make
+BuildRequires: pkgconfig(libei-1.0)
+BuildRequires: pkgconfig(libportal) >= 0.8
 %endif
 
 %if 0%{?suse_version}
-BuildRequires: avahi-compat-mDNSResponder-devel
+BuildRequires: pkgconfig(avahi-compat-libdns_sd)
 BuildRequires: cmake >= 3
 BuildRequires: desktop-file-utils
 BuildRequires: gcc-c++
-BuildRequires: gmock
-BuildRequires: gtest
-BuildRequires: libX11-devel
-BuildRequires: libXtst-devel libXinerama-devel libXrandr-devel
-BuildRequires: libICE-devel libSM-devel
-BuildRequires: libcurl-devel
-BuildRequires: libopenssl-devel
+BuildRequires: pkgconfig(gmock)
+BuildRequires: pkgconfig(gtest)
+BuildRequires: pkgconfig(x11)
+BuildRequires: pkgconfig(xtst) pkgconfig(xinerama) pkgconfig(xrandr)
+BuildRequires: pkgconfig(ice) pkgconfig(sm)
+BuildRequires: pkgconfig(libcurl)
+BuildRequires: pkgconfig(openssl)
 BuildRequires: qt6-base-devel
```

Comment:
I'm guessing this should be `BuildRequires: pkgconfig(Qt6)` as well?

Reply:
according to https://opensuse.pkgs.org/15.6/opensuse-oss-x86_64/qt6-base-devel-6.6.3-150600.1.18.noarch.rpm.html, only qt6-base-devel provides itself, not pkgconfig(Qt6)

Reply:
Maybe it could be replaced by cmake(Qt6)? This is the same in Fedora but qt6-base-common-devel (https://opensuse.pkgs.org/15.6/opensuse-oss-x86_64/qt6-base-common-devel-6.6.3-150600.1.18.aarch64.rpm.html) in SUSE

Reply:
IMO (but I'm not a maintainer here) it's easier to keep fedora/suse identical than worry about the most precise build requires (since they only apply to the build root anyway). So I'd say `pkgconfig(Qt6Platform)` is better - plus it looks more correct than that one random `-devel` package when everything else is using pkgconfig.

Reply:
I agree, I will try some things as soon as libportal 0.8 is in fedora

Reply:
Do you know anything about the difference Qt6Designer(Fedora) vs Qt6Linguist (SUSE)

Reply:
[virtually no knowledge of Qt]: I don't think they're the same, are they? One appears to be a design tool the other one for localization? If they show up as different on fedora/suse then maybe the deps of some other packages do or don't pull something in.

Reply:
pkgconfig(Qt6Platform) is not enough for SUSE, but pkgconfig(Qt6QuickWidgets) is for both (as in current commit)
pkgconfig(Qt6Linguist) is the same as pkgconfig(Qt6Linguist) in Fedora (but not in SUSE), so I now only use pkgconfig(Qt6Linguist)